docs/guestfs.md

```diff
@@ -0,0 +1,167 @@
+# Usage of libguestfs-tools and virtctl guestfs
+
+[Libguestfs tools](https://libguestfs.org/) are a set of utilities for accessing and modifying VM disk images. The command `virtclt guestfs` helps to deploy an interactive container with the libguestfs-tools and the PVC attached to it. This command is particurarly useful if the users need to modify, inspect or debug VM disks on a PVC.
+```bash
+$ virtctl guestfs -h
+Create a pod with libguestfs-tools, mount the pvc and attach a shell to it. The pvc is mounted under the /disks directory inside the pod for filesystem-based pvcs, or as /dev/vda for block-based pvcs
+
+Usage:
+  virtctl guestfs [flags]
+
+Examples:
+  # Create a pod with libguestfs-tools, mount the pvc and attach a shell to it:
+  virtctl guestfs <pvc-name>
+
+Flags:
+  -h, --help                 help for guestfs
+      --image string         libguestfs-tools container image
+      --kvm                  Use kvm for the libguestfs-tools container (default true)
+      --pull-policy string   pull policy for the libguestfs image (default "IfNotPresent")
+
+Use "virtctl options" for a list of global command-line options (applies to all commands).
+```
+
+The `virctl guestfs` sets up by default `kvm` for the interactive container. This considerably speeds up the execution of the libguestfs-tools since they use QEMU. If the cluster doesn't have any kvm supporting nodes, the user must disable kvm by setting the option `--kvm=false`. If not set, the libguestfs-tools pod will remain pending because it cannot be scheduled on any node.
+
+The command uses by default the image exposed by KubeVirt under the http endpoint `/apis/subresources.kubevirt.io/<kubevirt-version>/guestfs` but it can be configured to use a custom image by using the option `--image`. The users can also overwrite the pull policy of the image by setting the option `pull-policy`.
```

Comment:
The command uses by default the image exposed by KubeVirt under the http endpoint `/apis/subresources.kubevirt.io/<kubevirt-version>/guestfs` but it can be configured to use a custom image by using the option `--image` --> The command automatically uses the image exposed by KubeVirt under the http endpoint `/apis/subresources.kubevirt.io/<kubevirt-version>/guestfs`, but it can be configured to use a custom image by using the option `--image`.

The users --> Users